HdmiCecSink/HdmiCecSink.cpp

```diff
@@ -110,7 +113,8 @@ static char *eventString[] = {
         "arcTerminationEvent",
         "shortAudiodesciptorEvent",
         "standbyMessageReceived",
-        "setSystemAudioModeEvent"
+        "setSystemAudioModeEvent",
```

Comment:
reportAudioStatusEvent is defined in user sotry

Reply:
"setSystemAudioModeEvent",
        "reportAudioStatus"
};

this is from the above file